kamon-stackdriver/src/main/resources/reference.conf

```diff
@@ -1,5 +1,11 @@
 kamon {
   stackdriver {
+    tags {
```

Comment:
As this is used only for spans, maybe it makes sense to indicate that in config key name?

Reply:
Maybe move that next to `skip-operation-names`